test/unit/util_test.js

```diff
@@ -488,6 +488,37 @@ describe('Util', function ()
     }
   });
 
+  it('Util.url.appendRetryParam()', function () {
```

Comment:
Can you propose better title of this test?

Reply:
Changed.